partner_contact_tradename/models/res_partner.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    tradename = fields.Char(
+        size=300,
+    )
+
+    @api.model
+    def name_search(self, name="", args=None, operator="ilike", limit=100):
+        args = args or []
+        try:
+            recs = self.search(
```

Comment:
Should be good super call for other implementations of this feature or another modules

Reply:
Yes, will do the fix asap, thanks